packages/shared/src/components/ExtensionOnboarding.tsx

```diff
@@ -26,16 +22,15 @@ const ExtensionOnboarding = ({ onContinue }: Props): ReactElement => {
         The magic awaits inside! ✨
       </p>
 
-      {/* <Link href={onboardingUrl} passHref> */}
-      <Button
-        // tag="a"
-        className="z-1 w-full btn-primary max-w-[18.75rem]"
-        buttonSize={ButtonSize.Large}
-        onClick={() => onContinue(onboardingUrl)}
-      >
-        Continue ➔
-      </Button>
```

Comment:
Need to double check that this was not doing some extra things on `onContinue` which made the `Link` usage not an option

Reply:
checked, this was just a temporary change I made. `Link` is the way it was originally here.